catalog-info.yaml

```diff
@@ -1,13 +1,37 @@
+---
 apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
-  name: plugin-kafka-dev
-  description: An example of a Backstage application.
-  # Example for optional annotations
-  # annotations:
-  #   github.com/project-slug: backstage/backstage
-  #   backstage.io/techdocs-ref: dir:.
+  name: kafkaExample
+  title: Example For Kafka
+  description: |
+    Just a quick example
+
+  labels:
+    service: test
+    platform: example
+
+  tags:
+    - example
+
+  annotations:
+    # argocd/app-name: 'janus-idp'
+    kafka-info/consumer-groups: zomg
```

Comment:
This is good, but please add a comment to the README that mentions changing this to whatever a user would need.